rwatch/ui/action_menu.c

```diff
@@ -0,0 +1,266 @@
+/* menu.c
+ *
+ * Generic menu component.
```

Comment:
Can you improve this please

Reply:
I should clarify I meant the description of the component. "menu => action_menu. nice description of file